### PR TITLE
User input recipe count

### PR DESCRIPTION
**Pull request by AugustinasJarockis**
Added ability for the user to specify the amount of recipes to include in the site via command line or terminal. Renamed folder 'assets' to 'Images' and changed generator code in appropriate places.

### CODE REVIEW

File: generator/main.c [hunk 1]
```diff
@@ -23,19 +24,50 @@ void generate_main_page(const int recipe_count, char **recipe_titles, int *recip
...
+    {
+        return atoi(argv[1]);
+    }
+
+    char recipe_count[20];
+    do
+    {
+        printf("Number of recipes to include in the site:\n");
+        scanf("%19s", recipe_count);
+    } while(!is_integer(recipe_count) && (fflush(stdin), printf("Error: incorrect input. Please repeat the recipe count\n"), 1));
```
Comment (AugustinasJarockis) [inline generator/main.c:66]: I realise how dodgy this code is, but it works really nicely

**Review by AugustinasJarockis — COMMENTED** [review]
This one I just accidentally included unnecessary files

File: generator/main.c [hunk 2]
```diff
@@ -23,19 +24,50 @@ void generate_main_page(const int recipe_count, char **recipe_titles, int *recip
...
+    if(argc > 1 && is_integer(argv[1]))
+    {
+        return atoi(argv[1]);
+    }
+
+    char recipe_count[20];
+    do
+    {
+        printf("Number of recipes to include in the site:\n");
+        scanf("%19s", recipe_count);
```
Comment (arminasbrazenas) [inline generator/main.c:65]: Should we validate that we can actually generate such recipe count? 19 digits seems a lot too much, and we only have 7 of them... if somebody enters 10 it breaks?

Comment (AugustinasJarockis) [reply generator/main.c:65]: It is difficult to check how many files there are. If there are too few recipes the error will arise later

Comment (AugustinasJarockis) [reply generator/main.c:65]: And yes it breaks

Comment (AugustinasJarockis) [reply generator/main.c:65]: And I think it is the desired behaviour

**Pull request merged**